src/utils/queryBuilder.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+// Builds up filter string by filter type
+const eventFilters = new Map([
+  ['startDate', val => `start_date gt ${moment(val).format('YYYY-MM-DD')}`],
+  ['searchText', val => `contains(title, '${val}') or contains(name, '${val}') or contains(description, '${val}')`]
```

Comment:
I'm working off of the samples @aaghevli gave me and I think he's working off of [these](https://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part2-url-conventions/odata-v4.0-errata03-os-part2-url-conventions-complete.html#_Toc453752358) docs.